src/client/eclipse-map.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```

Comment:
I want Akos to be able to use it, so it should be accessible publicly and served through the webserver coz of the cross origin restriction